site/en/blog/immutable-document-domain/index.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: 'layouts/blog-post.njk'
+title: "`document.domain` is becoming immutable"
```

Comment:
Any ideas for a more descriptive title? Ideally, it would incorporate what `document.domain` enables--or rather what's becoming unavailable with this change. From what I gather, `document.domain` allowed same-origin policy relaxation [for specific use cases?], which will no longer be possible.

Reply:
How about "Relaxing same-origin policy by modifying `document.domain` will be disabled in Chrome"?

Reply:
cc: @lutzvahl

Reply:
Exactly what I was looking for! Here's the same thing reworded to match Google styleguide rules: "Chrome [version number] disables modifying `document.domain` to relax the same-origin policy".